src/ralph/export_to_ng/publishers.py

```diff
@@ -195,3 +201,81 @@ def sync_virtual_server_to_ralph3(sender, instance=None, created=False, **kwargs
         'parent_id': asset.id if asset else None,
         'custom_fields': _get_custom_fields(instance),
     }
+
+
+@ralph3_sync(Environment)
+def sync_network_environment_to_ralph3(sender, instance=None, created=False, **kwargs):
+    def convert_template(instance):
+        """Handle template without pipe inside"""
+        template = instance.hosts_naming_template
+        counter_result = re.search('<([0-9]+),([0-9]+)>', template)
+        if not counter_result:
+            logger.info(
+                'Incorrect template for network environment with id {}. Return default values.'.format(instance.id)  # noqa
+            )
+            return {
+                'hostname_template_prefix': '',
+                'hostname_template_counter_length': 4,
+                'hostname_template_postfix': instance.domain or ''
```

Comment:
maybe `'.{}'.format(instance.domain)` - in your case we'll have host `1234mydc.net`, assuming that domain name is `mydc.net`